VideoLocker/res/values/strings.xml

```diff
@@ -295,6 +295,12 @@
     <string name="discussion_priviledged_author_label_staff">Staff</string>
     <!-- The label to use when indicating a post was authored by a privileged user -->
     <string name="discussion_priviledged_author_attribution">By %s</string>
+    <!-- The label to use when describing the date and author of a discussion thread -->
+    <string name="discussion_thread_attribution">Discussion posted %1$s by %2$s %3$s</string>
+    <!-- The label to use when describing the date and author of a discussion question -->
+    <string name="question_thread_attribution">Question posted %1$s by %2$s %3$s</string>
+    <!-- The label to use when describing the date and author of a discussion comment -->
+    <string name="comment_attribution">Posted %1$s by %2$s %3$s</string>
```

Comment:
I tried using `Posted {time} by {author} {authorLabel}` but Phrase choked on it. So, leaving this how it is for now.